modules/core/src/main/java/org/apache/ignite/client/ClientCache.java

```diff
@@ -335,12 +337,19 @@
      */
     public <K1, V1> ClientCache<K1, V1> withKeepBinary();
 
+    /**
+     * Returns cache with the specified expired policy set. This policy will be used for each operation invoked on
+     * the returned cache.
+     *
+     * @return Cache instance with the specified expiry policy set.
+     */
+    public <K1, V1> ClientCache<K1, V1> withExpirePolicy(ExpiryPolicy expirePlc);
```

Comment:
I believe that this method should be consistent with a regular cache API and be named as "withExpir**y**policy". But since we are making a port from Ignite, let's keep the single code base for now.